### PR TITLE
Update CalDAV documentation

### PR DESCRIPTION
**Pull request by FuzzyMistborn**
Adding clarification that calendar and custom_calendar are exclusive, and also the ability to include all events in a custom_calendar

## Proposed change

Adding a cal_dav calendar was a bit frustrating as it wasn't clear that custom_calendar and calendar were mutually exclusive, and also it was unclear how to include all events in a calendar when using a custom calendar (ie using a wildcard).

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: N/A
- This PR fixes or closes issue: N/A

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_integrations/caldav.markdown [hunk 1]
```diff
@@ -79,13 +64,9 @@ calendar:
...
 
-This will create two binary sensors for the calendar name Agenda: "HomeOffice"
-and "WarmupFlat". Those sensors will be `on` if there is an ongoing event
-matching the regular expression specified in `search`.
-In custom calendars, events that last a whole day are taken into account.
+This will create two binary sensors for the calendar name Agenda: "HomeOffice" and "WarmupFlat". Those sensors will be `on` if there is an ongoing event matching the regular expression specified in `search`. In custom calendars, events that last a whole day are taken into account.
 
-Please note that when you configure custom calendars,
-the default ones are not created anymore.
+Please not that if you use the `custom_calendars` option, only those calendars will be loaded.  You cannot use `calendars` and `custom_calendars` in the same configuration.
```
Comment (frenck) [inline source/_integrations/caldav.markdown]:
```suggestion
Please note that if you use the `custom_calendars` option, only those calendars will be loaded. You cannot use `calendars` and `custom_calendars` in the same configuration.
```

File: source/_integrations/caldav.markdown [hunk 2]
```diff
@@ -102,13 +83,11 @@ password:
   type: string
 calendars:
   required: false
-  description: >
-    List of the calendars to filter.
-    Empty or absent means no filtering, i.e., all calendars will be added.
+  description: List of the calendars to filter. Empty or absent means no filtering, i.e., all calendars will be added. Cannot be used if custom_calendar option used
```
Comment (frenck) [inline source/_integrations/caldav.markdown]:
```suggestion
  description: List of the calendars to filter. Empty or absent means no filtering, i.e., all calendars will be added. It cannot be used if `custom_calendar` option used.
```

**Review by frenck — CHANGES_REQUESTED** [review]
Hi there @FuzzyMistborn, thanks for the PR.

I've left 2 comments, could you take a look?

Furthermore, I don't get why you changed all the markdown that isn't related to your PR at all. I would rather see you restore that. There is no need to change all that, secondly it made reviewing this PR really hard, because there wasn't obvious what was changed.

File: source/_integrations/caldav.markdown (hunk 1 above)
Comment (FuzzyMistborn) [reply source/_integrations/caldav.markdown]: Done

File: source/_integrations/caldav.markdown (hunk 2 above)
Comment (FuzzyMistborn) [reply source/_integrations/caldav.markdown]: Done

**Comment by FuzzyMistborn** [comment]
@frenck made the changes.  I didn't change any markdown.  Maybe this was by @klaasnicolaas with https://github.com/home-assistant/home-assistant.io/pull/12313/commits/a360f25419c25ab0747b9fbf2ea84bdef9aec33e ?

**Comment by klaasnicolaas** [comment]
@frenck that was me 😁

**Pull request merged**